packages/realm-react/package.json

```diff
@@ -17,7 +17,6 @@
       "command": "rollup --config",
       "dependencies": [
         "../realm:bundle",
-        "../realm:build:node",
```

Comment:
This would be a dependency for testing

Reply:
Yeah. We should probably add the `test` script to `wireit` too 👍

Reply:
It will happen soon, by me, when I get some realm/react work done